lib/algorithms/aes-gcm.js

```diff
@@ -326,10 +326,7 @@ var aesGcm = {};
 [
   "A128GCM",
   "A192GCM",
-  "A256GCM",
-  "A128GCMKW",
```

Comment:
I'd rather keep these in here, and rely on their removal from the supported algorithms list in `jwk/octkey.js`